html/webdriversessionsharing.html

```diff
@@ -11,44 +11,41 @@ <h1 class="page-header">Sharing WebDriver Sessions</h1>
 
   <h2>How to Share WebDriver Sessions Between Tests</h2>
   <ol>
-    <li>Annotate the test case with <code>@WebTest</code>. Give it a unique <code>sessionName</code> and set
-        <code>keepSessionOpen = true</code>.
-        This method is now the master test where it initiates the WebDriver session. You can now use the <code>sessionName</code>
-        in other tests within the class.
+    <li>Annotate the test class with <code>@Test</code> and <code>@WebTest</code>. Set the attribute
+        <code>singleThreaded = true</code>.
+        This class methods are executed sequentially with the same WebDriver session. You can now use the <code>priority</code>
+        attribute to order the test method execution sequence.
 
       <pre data-src="../sample-files/webdriversessionsharing-sample1.java"></pre>
      </li>
 
-    <li>To pick up the same test session in a different test, annotate the test with <code>@WebTest</code>
-        and use the same <code>sessionName</code> defined in the master test. Then, set  <code>keepSessionOpen = false</code>.
-        This will reuse the same session in this test instead of creating a new session. Since these tests depend on each
-        other, we can use TestNG's <code>dependsOnMethods</code> setting to make it run in
-        sequence. <p class="alert-info">Note: It's good practice to keep these tests in the same group.</p>
-
+     <li>To start test session in a test method, annotate the test with <code>@Test</code>
+        and use the attribute <code>priority = 1</code>.
 
       <pre data-src="../sample-files/webdriversessionsharing-sample2.java"></pre>
-      Here is a complete example that shows how a typical test would look:
+
+    <li>To pick up the same test session in a different test, annotate the test with <code>@Test</code>
+        and use the <code>priority</code> greater than defined in the master test. This will reuse the same session in this test 
+        instead of creating a new session. Since these tests order of execution controlled by TestNG priority.
+        <p class="alert-info">Note: It's good practice to keep these tests in the same group.</p>
 
       <pre data-src="../sample-files/webdriversessionsharing-sample3.java"></pre>
+      Here is a complete example that shows how a typical test would look:
+
+      <pre data-src="../sample-files/webdriversessionsharing-sample4.java"></pre>
       Web sessions can be shared to <var>n</var> <code>@Test</code> methods as long as test cases are running sequentially.
```

Comment:
Instead of telling Web Session sharing. Lets explain in general way ( Session Sharing)